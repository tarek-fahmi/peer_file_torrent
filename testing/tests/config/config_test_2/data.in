Invalid Packages Directory
./testing/bin/btide ./testing/resources/config/invalid_directoy.cfg