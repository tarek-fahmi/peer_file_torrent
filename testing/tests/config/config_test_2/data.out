Failed to load configuration: No such file or directory